index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="styles.css">
+    <title>TO-DO-List</title>
+</head>
+<body>
+    <header> To Do List
```

Comment:
```suggestion
    <header>
       <h1>To Do List</h1>
    </header>
```
In semantic HTML5, header elements should be used as a container for introductory content (ie h1-6 tags, etc).